tests/subsys/bluetooth/fast_pair/crypto/src/main.c

```diff
@@ -199,11 +199,23 @@ static void test_bloom_filter(void)
 	zassert_equal(s, sizeof(first_bloom_filter),
 		      "Invalid size of expected result.");
 	zassert_ok(fp_crypto_account_key_filter(first_result_buf, first_account_key_list,
-						ARRAY_SIZE(first_account_key_list), salt),
+						ARRAY_SIZE(first_account_key_list), salt, NULL),
 		   "Error during filter computing");
 	zassert_mem_equal(first_result_buf, first_bloom_filter, sizeof(first_bloom_filter),
 			  "Invalid resulting filter.");
 
+	static const uint8_t battery_info[] = {0b00110011, 0b01000000, 0b01000000, 0b01000000};
+
+	static const uint8_t first_bloom_filter_with_battery_info[] = {0x4A, 0x00, 0xF0, 0x00};
+
+	zassert_ok(fp_crypto_account_key_filter(first_result_buf, first_account_key_list,
+						ARRAY_SIZE(first_account_key_list), salt,
+						battery_info),
+		   "Error during filter computing");
+	zassert_mem_equal(first_result_buf, first_bloom_filter_with_battery_info,
```

Comment:
Should we also check that `sizeof(first_result_buf) >= sizeof(first_bloom_filter_with_battery_inf)`?

Reply:
yes, we can do it. We should actually check if the sizes are equal.